composer.json

```diff
@@ -3,7 +3,7 @@
   "autoload": {
     "psr-0": {
       "SprykerEco": "src/",
-      "Function": "tests/"
+      "SprykerEcoTest": "tests/"
```

Comment:
No more psr0 please, only psr4

Reply:
Split into autoload and autoload-dev and changed psr-0 to par-4